OpenVPNHelper/OpenVPNHelper.m

```diff
@@ -85,6 +85,43 @@ - (BOOL)verify:(NSString *)identifier atURL:(NSURL *)fileURL {
 }
 
 - (void)startOpenVPNAtURL:(NSURL *_Nonnull)launchURL withConfig:(NSURL *_Nonnull)config upScript:(NSURL *_Nullable)upScript downScript:(NSURL *_Nullable)downScript leasewatchPlist:(NSURL *_Nullable)leasewatchPlist leasewatchScript:(NSURL *_Nullable)leasewatchScript scriptOptions:(NSArray <NSString *>*_Nullable)scriptOptions reply:(void(^_Nonnull)(BOOL))reply {
+    
+    // Get the path of config file
+    NSString* path = config.path;
+    NSString* content = [NSString stringWithContentsOfFile:path
+                                                  encoding:NSUTF8StringEncoding
+                                                     error:NULL];
+    
+    NSMutableArray *listItems = [content componentsSeparatedByString:@"\r\n"];
+    NSArray *maliciousCommands = @[@"up", @"tls-verify", @"ipchange", @"client-connect", @"route-up",@"route-pre-down",@"client-disconnect",@"down",@"learn-address",@"auth-user-pass-verify"];
```

Comment:
Not sure, but is an openvpn command case sensitive?

Reply:
Good point , I have fixed that